Aptoide/appcoins-wallet-android
# Feature/mob 629 gms rakam property

**What does this PR do?**

This event implements a new super property to rakam to identify users without GMS;

**Database changed?**

No

**Where should the reviewer start?**

- [ ] RakamAnalytics.kt

**How should this be manually tested?**

To test this you need to check rakam and make sure the events are getting there correctly. You can speak to me if you need help doing so;

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-629](https://aptoide.atlassian.net/browse/MOB-629)

**Code Review Checklist**
